client/src/itemDisplay.tsx

```diff
@@ -145,69 +137,101 @@ export const ItemDisplay = ({
       >
         <div
           css={css`
+            position: relative;
             color: ${palette.neutral["20"]};
             ${agateSans.xxsmall({ lineHeight: "tight" })};
             margin-bottom: 2px;
           `}
         >
           <FormattedDateTime timestamp={dateInMillisecs} />
+          {isEdited && (
+            <span>
+              &nbsp;-&nbsp;<em>Edited</em>
+            </span>
+          )}
+          {!isDeleted &&
```

Comment:
there are a lot of conditions here, maybe extract to a meaningful const like `isEditableItem` or similar

Reply:
good suggestion, thanks - done in https://github.com/guardian/pinboard/pull/230/commits/42cd458c05a1a9f2f540c034a220d9747d724391